tests/testthat/test-validate.R

```diff
@@ -45,6 +45,19 @@ test_that("validate count matrix", {
   expect_false(resp$success)
   expect_match(resp$msg, "should be unique")
 
+  # empty barcode
+  mat <- create_count_mat(3, 3)
+  colnames(mat) <- c("ACTGAA-1", "ACTGAA-1", "")
+  resp <- validate_count_mat(mat)
+  expect_false(resp$success)
+  expect_match(resp$msg, "cannot be the empty string")
+
+  # empty feature
+  mat <- create_count_mat(3, 3)
+  colnames(mat) <- c("ACTGAA-1", "ACTAAA-1", "")
+  resp <- validate_count_mat(mat)
+  expect_false(resp$success)
+
```

Comment:
since`colNames` contains empty string wouldn't this fail `barcodes cannot be the empty string` instead of `features cannot be the empty string`